docs/examples/SPA.md

```diff
@@ -0,0 +1,66 @@
+## Using offline-plugin in a Single Page Application (SPA)
+
+In a SPA, only the inital HTML is served from the server. Any routing beyond that takes place on the client. So how should we configure offline-plugin to handle this?
```

Comment:
It's much weirded to refer to that as `/app-shell.html` though. `/` is fine as you did. After, we aren't here to help people learn how to do SPA or SPA with server rendering. They at least should understand that they need to cache some html (app-shell) for all routes for their front-end to start working. If they don't understand how it works in general (e.g. like recent events with multi accounts) then they should learn that first.